PAS_Database/dbo/Stored Procedures/usprpt_GetWorkOrderAwaitingPartsReport.sql

```diff
@@ -106,33 +114,35 @@ BEGIN
 		IF OBJECT_ID(N'tempdb..#tmpMultipleWOMStocklineKit') IS NOT NULL
 			DROP TABLE #tmpMultipleWOMStocklineKit
 
-		IF OBJECT_ID(N'tempdb..#tmpMultipleSubWOMStockline') IS NOT NULL
-			DROP TABLE #tmpMultipleSubWOMStockline
-
-		IF OBJECT_ID(N'tempdb..#tmpMultipleSubWOMStocklineKit') IS NOT NULL
-			DROP TABLE #tmpMultipleSubWOMStocklineKit
-
 		CREATE TABLE #AwaitingPartsData
 		(	
 			TotalRecordsCount BIGINT NULL, 
 			WorkOrderId BIGINT NULL, 
 			wonum VARCHAR(50) NULL, 
 			customername VARCHAR(500) NULL, 
 			woqnum VARCHAR(50) NULL, 
+			mpn VARCHAR(500) NULL, 
+			mpnDescription VARCHAR(500) NULL,
+			itemMasterId BIGINT NULL,
 			pn VARCHAR(500) NULL, 
 			pnDescription VARCHAR(500) NULL,
```

Comment:
This is also needs to be change as same as master table

Reply:
Resolved